cmd/manager/main.go

```diff
@@ -75,9 +76,12 @@ func main() {
 	// TODO: resyncInterval is hardcoded to 1 hour now, it would have to be
 	// configurable on a per OperatorSource level.
 	initialWait, resyncInterval := time.Duration(10), time.Duration(60)
-	syncer := operatorsource.NewRegistrySyncer(mgr.GetClient(), initialWait, resyncInterval)
 
-	go syncer.Sync(stopCh)
+	catalogSyncer := catalogsourceconfig.NewCatalogSyncer(mgr.GetClient(), initialWait)
+	registrySyncer := operatorsource.NewRegistrySyncer(mgr.GetClient(), initialWait, resyncInterval, catalogSyncer)
```

Comment:
Do we want both of these to have the exact same initialWait? Wouldn't it be better to stagger them slightly? I think in the current implementation it shouldn't matter, but there could be some timing issues if these get more complex.

Reply:
catalogSyncer waits for an event from registrySyncer, so it already is staggered in that regard. But the larger timing issue as we discussed earlier still remains with this approach.

Reply:
fair